userguide/sharing.rst

```diff
@@ -920,7 +920,10 @@ provides more details for each configurable option.
    |                                |               |          |                                                                                                                                                      |
    +--------------------------------+---------------+----------+------------------------------------------------------------------------------------------------------------------------------------------------------+
    | Export Recycle Bin             | checkbox      | ✓        | Set for deleted files to move to :file:`.recycle` in the root folder of the share. The :file:`.recycle` directory can be deleted to reclaim space    |
-   |                                |               |          | and is recreated whenever a file is deleted.                                                                                                         |
+   |                                |               |          | and is recreated whenever a file is deleted. If a file being deleted is in a subdirectory and larger than 20 MiB, the file is deleted permanently.   |
+   |                                |               |          | To move files larger than 20 MiB into :file:`.recycle`, adjust the :guilabel:`Auxiliary Parameter` :command:`crossrename:sizelimit=` to a            |
```

Comment:
I'd explain this as "To allow larger files to be moved to the Recycle Bin rather than deleted immediately, adjust the ... to a larger size.  For example..."